### PR TITLE
Manually map models on migration instead of loading via directory

### PR DESCRIPTION
**Pull request by alecritson**
Currently if you publish the migrations they can fail due to trying to read a directory relative to the the vendor folder. This PR instead opts to manually map the models to prevent this issue completely.

### CODE REVIEW

File: packages/core/database/migrations/2024_03_15_100000_remap_polymorphic_relations.php [hunk 1]
```diff
@@ -1,20 +1,56 @@
...
-use Lunar\Base\BaseModel;
 use Lunar\Base\Migration;
-use Spatie\StructureDiscoverer\Discover;
 
 class RemapPolymorphicRelations extends Migration
 {
     public function up()
     {
-        $modelClasses = collect(
-            Discover::in(__DIR__.'/../../src/Models')
```
Comment (wychoong) [inline packages/core/database/migrations/2024_03_15_100000_remap_polymorphic_relations.php:13]: what about `base_path('vendor/lunarphp/core/src/Models')`
https://laravel.com/docs/11.x/helpers#method-base-path

Comment (glennjacobs) [reply packages/core/database/migrations/2024_03_15_100000_remap_polymorphic_relations.php:13]: Yeah, I would have taken that approach also.

Comment (alecritson) [reply packages/core/database/migrations/2024_03_15_100000_remap_polymorphic_relations.php:13]: I tried this but it wouldn't work for the testsuite as Orchestra loads an instance of a Laravel app within itself so the base path would be relative to that install, which doesn't have a vendor directory

Comment (glennjacobs) [reply packages/core/database/migrations/2024_03_15_100000_remap_polymorphic_relations.php:13]: I guess it's only a migration, not something that will need to be maintained. So it's fine.

Comment (alecritson) [reply packages/core/database/migrations/2024_03_15_100000_remap_polymorphic_relations.php:13]: Can you clarify what's fine, the proposed change here or the current change in the PR?

Comment (glennjacobs) [reply packages/core/database/migrations/2024_03_15_100000_remap_polymorphic_relations.php:13]: I'm happy with the PR as it is. If the base_path() won't work, then I don't think we worry about it.

**Review by glennjacobs — APPROVED** [review]

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**